docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2718,7 +2718,7 @@ definitions:
           structure is not enforced. For example a team name and data type can be used such as
```

Comment:
Maybe also remove "team name can be used"?  Let's not recommend that (or change the wording to the opposite).

Reply:
:+1: